keras_cv/layers/object_detection/roi_align.py

```diff
@@ -17,11 +17,13 @@
 from typing import Optional
 from typing import Tuple
 
+# TODO (ariG23498): remove tf and correct the type imports
```

Comment:
Can this be removed now:

Reply:
Could you point me to the way we type hint keras code now? I can go ahead and make the changes and remove the import.

Reply:
We generally don't use typehints, but instead document expected types in docstrings.